frontend/src/pages/Match/index.js

```diff
@@ -33,29 +33,25 @@ function MatchPage() {
   const alertCtx = useContext(AlertContext);
   const userCtx = useContext(UserContext);
   // Difficulty
-  const [difficulty, setDifficulty] = useState('');
+  const difficulty = location.state?.difficulty;
   // Socket
-  const [socket, setSocket] = useState(null);
+  const socket = io(URL_MATCHING_SVC_CONNECT, {
+    path: '/api/matching-service/socket',
```

Comment:
Please put this hardcoded path into config.js, thanks! Let's consolidate all the URL into the file. 😄

Reply:
Agree